### PR TITLE
[GH-877] Change dev input bucket location

### PR DESCRIPTION
**Pull request by jack-r-warren**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-877
- https://broadinstitute.atlassian.net/browse/GH-873
- Consequence of https://github.com/broadinstitute/gotc-deploy/pull/114

### Changes

- Change bucket location and WFL URL to what they will be

### Review Instructions

- We'll be able to deploy the cloud function here as soon as the move is complete

### CODE REVIEW

File: cloud_function/deploy.sh [hunk 1]
```diff
@@ -6,10 +6,10 @@ TRIGGER_BUCKET=${2}
 TRIGGER_EVENT=${3:-"google.storage.object.finalize"}
 REGION=${4:-"us-central1"}
 
-if [ "${GCLOUD_PROJECT}" == "broad-gotc-dev" ]; then
+if [ "${GCLOUD_PROJECT}" == "broad-gotc-dev-storage" ]; then
   # This service account must have access to the WFL API
   SA_EMAIL="aou-cloud-fn-non-prod@broad-gotc-dev.iam.gserviceaccount.com"
```
Comment (samanehsan) [inline cloud_function/deploy.sh:11]: This service account should be "aou-cloud-fn-non-prod@broad-gotc-dev-storage.iam.gserviceaccount.com" now!

Comment (jack-r-warren) [inline cloud_function/deploy.sh:11]:
```suggestion
  SA_EMAIL="aou-cloud-fn-non-prod@broad-gotc-dev-storage.iam.gserviceaccount.com"
```
Email got updated via https://github.com/broadinstitute/gotc-deploy/pull/115

Comment (jack-r-warren) [reply cloud_function/deploy.sh:11]: Omg we both made the same comment at the same time

Comment (samanehsan) [reply cloud_function/deploy.sh:11]: 😆

**Pull request merged**